ITEM_en.tsv

```diff
@@ -110,11 +110,11 @@ ITEM_20150317_000120	This blade severely threatens both the life of the foe and
 ITEM_20150317_000121	Scheduled for deletion
 ITEM_20150317_000122	Two-Handed Sword [Slash]
 ITEM_20150317_000123	Dunkel Bastard Sword
-ITEM_20150317_000124	Some master craftsman and soldiers call this a 'mixed style but a two-handed sword.' They say this to justify themselves.
+ITEM_20150317_000124	Some master craftsman and soldiers call this a 'mixed style two-handed sword.' They say this to justify themselves.
```

Comment:
This sounds better, but technically in the Korean it has the conjunction in there.